### PR TITLE
[bitnami/harbor-notary-*] Deprecate Harbor Notary Signer and Harbor Notary Server

### PR DESCRIPTION
**Pull request by Mauraza**
Deprecate Harbor Notary Signer and Harbor Notary Server

### CODE REVIEW

**Review by dgomezleon — DISMISSED** [review]

File: config/components/harbor-notary-server.json [hunk 1]
```diff
@@ -1,3 +1,4 @@
 {
-  "name": "harbor-notary-server"
+  "name": "harbor-notary-server",
+  "to-be-deprecated": "20230914"
```
Comment (gongomgra) [inline config/components/harbor-notary-server.json:3]: Deprecation date should be `20231014`

File: config/components/harbor-notary-signer.json [hunk 2]
```diff
@@ -1,3 +1,4 @@
 {
-  "name": "harbor-notary-signer"
+  "name": "harbor-notary-signer",
+  "to-be-deprecated": "20230914"
```
Comment (gongomgra) [inline config/components/harbor-notary-signer.json:3]: The same

**Review by gongomgra — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -127,6 +127,11 @@ $ docker run bitnami/postgresql cat /opt/bitnami/postgresql/.spdx-postgresql.spd
 * Wavefront Prometheus Adapter
 * Wavefront HPA Adapter
 
+### 2023-09
+
+* Harbor Notary signer
+* Harbor Notary server
```
Comment (carrodher) [inline README.md:133]: This should be placed on top of the list, months are listed from recent to old

Comment (Mauraza) [reply README.md]: Update

File: config/components/harbor-notary-signer.json (hunk 2 above)
Comment (Mauraza) [reply config/components/harbor-notary-signer.json]: Update

**Review by carrodher — APPROVED** [review]

**Review by gongomgra — APPROVED** [review]
LGTM!

**Pull request merged**